samples/excel/45-named-item/create-and-use-named-item-for-range.yaml

```diff
@@ -0,0 +1,180 @@
+id: excel-create-and-use-named-range-item
+name: Create and use named range item
+description: Create and use a named range item
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |-
+        $("#add-name").click(() => tryCatch(addName));
+        $("#format-named-range").click(() => tryCatch(formatRangeByName));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function addName() {
+            await Excel.run(async (context) => {
+
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const headerRange = sheet.getRange("A1:E1");
+
+                sheet.names.add("ExpensesHeader", headerRange);
+
+                const namedItems = sheet.names.load();
+
+                await context.sync();
+
+                console.log("This workbook contains " + namedItems.items.length + " named items.");
```

Comment:
It would be good to start using OfficeHelpers.UI.notify more.  At some point we may try to make the console less prominent (since folks often minimize it anyway).  In fact, I even have an issue to remove `console.log` from older samples; let's not add to the backlog of them.  https://github.com/OfficeDev/office-js-snippets/issues/28